cli/args/flags.rs

```diff
@@ -3537,6 +3631,21 @@ fn runtime_args(
     .arg(strace_ops_arg())
 }
 
+fn allow_import_arg() -> Arg {
+  Arg::new("allow-import")
+    .long("allow-import")
+    .short('I')
+    .num_args(0..)
+    .use_value_delimiter(true)
+    .require_equals(true)
+    .value_name("IP_OR_HOSTNAME")
+    .help(concat!(
+      "Allow importing from remote hosts. Optionally specify allowed IP addresses and host names, with ports as necessary. ",
+      "Allowed by default: deno.land, jsr.io, esm.sh, raw.githubusercontent.com, user.githubusercontent.com"
```

Comment:
Mention that pass value will override default hosts

Reply:
Changing "Alowed by default:" to "Default value:"